### DISCUSSION TITLE
Serve files using express

### DISCUSSION THREAD

**Discussion by YutongZhuu** [jeffreylanters/react-unity-webgl#534] [Feature Requests]
Hi, I made my project work by serving Unity static files via express. A lot of people who are new to Web/Unity, like me, might not know this option, I want to add a few sentences in the HelloWorld.md to tell people this was an option too, (and I think moving large files to the backend is good practice?)

**Comment by raphaelbp12** [comment]
hello @YutongZhuu ! How did you referenced the files URL to the `useUnityContext`? Did you simply put the whole URL there? Can you please share a bit your solution?

**Reply by YutongZhuu** [reply]
I think there were four files, and I used the express app to serve them as static content. And I do not remember how exactly I referenced them (but i think it is just how you normally access a static file),  it was a year ago, sorry about that.
